configs/benchmark/values.yaml

```diff
@@ -0,0 +1,8 @@
+wrk2:
+  duration: 120
+  connections: 96
+  RPS: 3000
+  serviceMesh: "istio"
```

Comment:
Can we leave this variable empty? Otherwise runs where `wrk2.serviceMesh` was not set explicitly will default to "istio" while these really should default to "bare-metal".